inventree/part.py

```diff
@@ -111,6 +111,20 @@ def uploadImage(self, image):
 
         return response
 
+    def getInternalPrice(self):
```

Comment:
I think this method would be better named either:

- `getInternalPrices`
- `getInternalPriceList`

Reply:
Noted